ci/run_easymesh_cert.sh

```diff
@@ -22,7 +22,10 @@ usage() {
     echo "      -b|--branch - prplmesh branch to use (default master latest)"
     echo "      -o|--log-folder - path to put the logs to (make sure it does not contain previous logs)."
     echo "      -e|--easymesh-cert - path to easymesh_cert repository (default ../easymesh_cert)"
-    echo "      -u|--upload - upload results"
+    echo "      --owncloud-upload - weather or not to upload results to owncloud"
+    echo "      --owncloud-user - owncloud username"
+    echo "      --owncloud-password - owncloud password"
+    echo "      --owncloud-path - relative path in the owncloud server to upload results to"
```

Comment:
Please mention the default. Note that you can use 
```suggestion
    echo "      --owncloud-path - relative path in the owncloud server to upload results to"
    echo "                                   (default: $OWNCLOUD_PATH)"
```